aten/src/ATen/native/mkldnn/xpu/detail/QMatmul.cpp

```diff
@@ -0,0 +1,281 @@
+#include <ATen/Tensor.h>
+#include <ATen/core/Tensor.h>
+#include <c10/core/ScalarType.h>
+
+#include <ATen/native/mkldnn/xpu/detail/Attr.h>
+#include <ATen/native/mkldnn/xpu/detail/oneDNNContext.h>
+
+#include <oneapi/dnnl/dnnl.hpp>
+
+namespace at::native::onednn {
+
+void quantized_matmul(
+    at::Tensor mat1, // act
+    double input_scale,
+    int64_t input_zero_point,
+    at::Tensor mat2, // weight
+    at::Tensor& weight_scales,
+    at::Tensor& weight_zero_points,
+    at::Tensor& b_raw,
```

Comment:
Should it be `std::optional`?

Reply:
Thanks for reminding, I check my code, there should be no `c10::optional` in my code now. `std::optional` is recommended at present.